src/components/ui/QualitityControl.jsx

```
@@ -0,0 +1,30 @@
+import { motion } from "motion/react";
```

ESLint

[error] 'motion' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)